### ISSUE TITLE
Failed to get ANSI buffer size(WideCharToMultiByte: El parßmetro no es correcto. )Failed to convert pyhome to ANSI (invalid multibyte string)

### ISSUE THREAD

**Issue by ruverav** [docker/compose#2779]
Not sure if it's my fault but docker-compose has stopped working for me on Windows 10 (64 bits) with Docker Toolbox.

I normally use it from MINGW64, but whenever I type it (I have also tried on cmd.exe) I get the following error:

```
$ docker-compose
Failed to get ANSI buffer size(WideCharToMultiByte: El parßmetro no es correcto.
)Failed to convert pyhome to ANSI (invalid multibyte string)
```

> NOTE:  "El parßmetro no es correcto." means "Parameter is not correct." in spanish.

I have been using it without issues or warnings until yesterday, but suddenly stopped working.

I have also downloaded the latest version (1.6.0) for Windows but still does not work. I don't know what else to do but I urgently need it to work.

Thanks for your help and advices.

**Comment by ruverav** [comment]
I installed [Python 3.4.3](https://www.python.org/downloads/release/python-343/), [pip](https://bootstrap.pypa.io/get-pip.py) added `C:\Python34` and `C:\Python34\Scripts` to `PATH `and copied `C:\Python34\Scripts\docker-compose.exe` to `C:\Program Files\Docker Toolbox`. Now it works thanks to Python interpreter.

It seems my problem is related to how Windows binaries are built when packaged with Docker Toolbox. [This post on Stackoverflow](http://stackoverflow.com/questions/34449674/exe-file-produced-by-pyinstaller-cannot-be-used-in-others-pc) could shed more light about the problem's

**Comment by dnephin** [comment]
Thanks for trying that out. I had a similar suspicion that using python 3 would fix it. 

I spent a few minutes trying to get our binary built using python 3 on appveyor, but I was running into errors with pyinstaller. I'll have to look into it again later and see if we can change to python3 by default.

**Comment by ruverav** [comment]
Thanks for your response. I have zero knowledge about Python and no time apart from work to try to compile it my self, but I hope it can be easily fixed using a newer version of Python.

**Comment by shin-** [comment]
This was likely fixed by #2888
